tests/dummy/app/pods/docs/guides/avoiding-rendering-errors/template.md

```diff
@@ -22,4 +22,32 @@ If this template is rendered with a `post` that has not loaded its `comments.aut
 
 This template assertion is especially useful for reusable components with complex data requirements.
 
-Async relationships can also lead to surprises in templates by adding unnecessary states to your application. Read the section "Avoid async relationships" from the previous guide to learn more about their pitfalls, and how to enforce sync-only relationships in your apps.
+Async relationships can also lead to surprises in actions by adding unnecessary states to your application. Read the section "Avoid async relationships" from the previous guide to learn more about their pitfalls, and how to enforce sync-only relationships in your apps.
+
+# Ensure data is loaded at run time
```

Comment:
This is a new H1 heading but let's make it H2. And let's make the two H2s on this page something like:

- Ensuring data is loaded within templates
- Ensuring data is loaded within JavaScript files